modules/preparing-an-inital-cluster-deployment-for-mce.adoc

```diff
@@ -0,0 +1,401 @@
+<3> Set the registry URL for the storage backend.
+<4> Set the channel to retrieve the {product-title} images from.
+<5> Set the Operator catalog to retrieve the {product-title} images from.
+<6> Specify only certain Operator packages and channels to include in the image set. Remove this field to retrieve all packages in the catalog.
+<7> The MCE Operator packages and channels.
+<8> The LSO Operator packages and channels.
+
+. To mirror the specific {product-title} image repository, the MCE Operator and Local Storage Operator(LSO), run the following command:
+
++
+[source,terminal]
+----
+oc mirror --dest-skip-tls --config ocp-mce-imageset.yaml docker://<your-local-registry-dns-name>:<your-local-registry-port>
+----
+
+
+. Update the registry and certificate in the `install-config.yaml` file:
+
++
+.Example `imageContentSources.yaml`
+
+[source,yaml]
+----
+  imageContentSources:
+    - source: "quay.io/openshift-release-dev/ocp-release"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/openshift/release-images"
+    - source: "quay.io/openshift-release-dev/ocp-v4.0-art-dev"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/openshift/release"
+    - source: "registry.redhat.io/ubi8"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/ubi8"
+    - source: "registry.redhat.io/multicluster-engine"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/multicluster-engine"
+    - source: "registry.redhat.io/rhel8"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/rhel8"
+    - source: "registry.redhat.io/redhat"
+      mirrors:
+        - "<your-local-registry-dns-name>:<your-local-registry-port>/redhat"
+----
+
++
+Additionally, ensure your certificate is present in the `additionalTrustBundle` field of the `install-config.yaml`.
+
++
+.Example `install-config.yaml`
+[source,yaml]
+----
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  . zzzzzzzzzzz
+  .  -----END CERTIFICATE-----
+----
+
++
+[IMPORTANT]
+====
+The `oc mirror` command  creates a folder called **oc-mirror-workspace** with several outputs.
+This includes the `imageContentSourcePolicy.yaml` file that identifies all the mirrors you need for {product-title}  and your selected Operators.
+====
+
+. Generate the cluster manifests by running the following command:
+
++
+[source,terminal]
+----
+$ openshift-install agent create cluster-manifests
+----
+This command updates the `cluster manifests` folder to include a `mirror` folder that contains your mirror configuration.
+
+. Create a sub-folder named  `openshift` in the `<manifests_directory>` folder. This sub-folder is used to store the extra manifests that will be applied during the installation to further customize the deployed cluster.
+
+. For MCE, create the following manifests and save in the `<manifests_directory>/openshift` folder:
```

Comment:
It would be nice to briefly introduce an high level overview of the procedure, and in particular which operators we're going to install.